views/layout.pug

```diff
@@ -77,19 +77,25 @@ html(lang='en', prefix='og: http://ogp.me/ns#', itemscope, itemtype='http://sche
                         .col-sm-12.d-none.d-lg-block
                             .twitter-timeline-custom
 
-        script(src=config.javascript.jquery.uri,
-            integrity=config.javascript.jquery.sri, crossorigin='anonymous')
-
         -var bootstrap = config.bootstrap4.filter(function(o) { return o.latest; })[0];
-        script(src=bootstrap.javascriptBundle, integrity=bootstrap.javascriptBundleSri,
-            crossorigin='anonymous')
 
         script(nonce=nonce)
-            include /assets/js/clipboard.min.js
+            include /assets/js/vendor/clipboard.min.js
+            include /assets/js/vendor/loadjs.min.js
 
             if (process.env.NODE_ENV === 'production')
                 include:uglify-js /assets/js/main.js
             else
                 include /assets/js/main.js
 
+            | loadjs(['#{config.javascript.jquery.uri}', '#{bootstrap.javascriptBundle}'], 'jquery', {
+            |     async: false,
+            |     numRetries: 3
+            | });
+            | loadjs.ready('jquery', {
+            |     success: function() {
```

Comment:
Could probably just make this...

```javascript
loadjs.ready('jquery', {
    success: $('.ads-info-toggler').popover
});
```

Reply:
True, changed it, and move the whole block to a pug include. The jQuery code is still hardcoded, not sure if it's worth having it separate since it's just one line.

Reply:
Yeah, that's what I was thinking. Not really worth it.